docs/docs/api/rest/workflows/README.md

```diff
@@ -0,0 +1,247 @@
+
+The *Slack - Send message in a Public Channel* step requires a `channelId`, `message` and the connected Slack account (`slack`). Let's start with connecting the Slack account.
+
+#### Find your connected accounts
+
+To connect your accounts to the workflow, you'll need to find the specific IDs for each of the accounts you'd like to connect.
+
+You can find your connected account IDs by using the [List Accounts endpoint](/api/rest/#get-workspaces-s-connected-accounts).
+
+You can filter your accounts by using the `app_slug` query parameter. For example, if you want to find your connected Slack accounts to your workspace, then add `slack` to the query param:
+
+```
+GET /workspaces/<workspace_id>/accounts?app_slug=slack
+```
+
+This request will narrow down the results to only your connected Slack accounts, for easier finding.
+
+You'll need the ID of each connected account you'd like to configure this new workflow with. Copy down the `apn_******` value of the connected accounts from the response you'd like to use for the steps.
+
+```json{10}
+{
+  "page_info": {
+    "total_count": 1,
+    "count": 1,
+    "start_cursor": "YXBuXzJrVmhMUg",
+    "end_cursor": "YXBuXzJrVmhMUg"
+  },
+  "data": [
+    {
+      "id": "apn_abc123",
+      "name": "Slack Pipedream Workspace"
+    }
+  ]
+}
+```
+
+Now we can copy the ID for our Slack account from the response: `apn_abc123`.
+
+Given we now have the connected account ID, we can design the rest of the payload:
+
+```json{13-20}
+{
+  "project_id": "proj_abc123",
+  "org_id": "o_abc123",
+  "template_id": "tch_3BXfWO",
+  "triggers": [
+    {
+      "props": {
+        "url": "https://hnrss.org/newest?q=Pipedream"
+      }
+    }
+  ],
+  "steps": {
+    {
+      "namespace": "send-message",
+      "props": {
+        "slack": "apn_abc123",
+        "channelId": "U12356",
+        "message": "**New HackerNews Mention** \n \n {{steps.trigger.event.item.title}} \n {{steps.trigger.event.item.description}}"
+      }
+    }
+  }
+}
+```
+Our payload now instructs Pipedream to set up the `send-message` step in our workflow with our connected Slack account and specific `channelId` and `message` parameters.
+
+### Step 4 - Define settings (optional)
+
+You can also define workflow settings such as the workflows, name, allocated memory, or if it should be deployed immediately:
+
+```json{22-25}
+{
+  "project_id": "proj_abc123",
+  "org_id": "o_abc123",
+  "template_id": "tch_3BXfWO",
+  "triggers": [
+    {
+      "props": {
+        "url": "https://hnrss.org/newest?q=Pipedream"
+      }
+    }
+  ],
+  "steps": {
+    {
+      "namespace": "send-message",
+      "props": {
+        "slack": "apn_abc123",
+        "channelId": "U12356",
+        "message": "**New HackerNews Mention** \n \n {{steps.trigger.event.item.title}} \n {{steps.trigger.event.item.description}}"
+      }
+    }
+  },
+  "settings": {
+    "name": "New HackerNews Mentions to Slack",
+    "autodeploy": true
```

Comment:
This param is currently called `auto_deploy`. Should it be changed to `autodeploy` in WorkflowsController?

```suggestion
    "auto_deploy": true
```